tests/unit/util/async/AnyExecutionContextTests.cpp

```diff
@@ -270,6 +273,20 @@ TEST_F(AnyExecutionContextTests, TimerWithBoolHandlerExecuted)
     EXPECT_EQ(timer.get().value(), 42);
 }
 
+TEST_F(AnyExecutionContextTests, RepeatingOperation)
+{
+    auto mockRepeatingOp = RepeatingOperationType<std::any>{};
+    EXPECT_CALL(mockRepeatingOp, wait());
+    EXPECT_CALL(
+        mockExecutionContext, executeRepeatedly(An<std::chrono::milliseconds>(), An<std::function<std::any()>>())
+    )
+        .WillOnce([&mockRepeatingOp] -> RepeatingOperationType<std::any> const& { return mockRepeatingOp; });
+
+    auto res = ctx.executeRepeatedly(std::chrono::milliseconds(1), [] -> void { throw 0; });
```

Comment:
Why throwing? If you don't want this to be called probably strict function mock would be better.

Reply:
It's just a way to make sure this function does not actually gets called or used - that's not the intention of this test. it's the same with all other checks in this file so i just went with the same style